### PR TITLE
Bs 61

### LINKED ISSUE #61: создание Subrcription UI

### PR DESCRIPTION
**Pull request by antonhoang**
## Details
- implementing subscription UI

## Screenshots 
Screenshot 2022-08-07 at 11 49 04

### CODE REVIEW

**Comment by VladimirBrejcha** [comment]
image

Looks like overlapping safe area on the top

File: bonsai/Modules/Subscriptions/Subscriptions.swift [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                .listRowSeparator(.hidden)
+                .listRowBackground(BonsaiColor.back)
+                
+                HStack(alignment: .center) {
+                    Spacer()
+                    ZStack {
+                        RoundedRectangle(cornerRadius: 13)
+                            .frame(width: 192, height: 48)
+                            .foregroundColor(BonsaiColor.mainPurple)
+                        Text("Get all feutures")
```
Comment (VladimirBrejcha) [inline bonsai/Modules/Subscriptions/Subscriptions.swift]: typo `features`

**Comment by VladimirBrejcha** [comment]
I think Best Value subscription should be selected initially

File: bonsai/Modules/Subscriptions/Subscriptions.swift (hunk 1 above)
Comment (antonhoang) [reply bonsai/Modules/Subscriptions/Subscriptions.swift]: updated+

**Comment by antonhoang** [comment]
> I think Best Value subscription should be selected initially

updated+

**Comment by antonhoang** [comment]
> image
> 
> Looks like overlapping safe area on the top

updated+

**Pull request merged**